sources/packages/backend/workflow/test/2021-2022/fulltime-assessment/eligibility/fulltime-assessment-2021-2022-eligibility-BCAG.e2e-spec.ts

```diff
@@ -21,10 +21,7 @@ describe(`E2E Test Workflow fulltime-assessment-${PROGRAM_YEAR}-eligibility-BCAG
     CredentialType,
   ).filter((type) => !EXPECTED_PROGRAM_CREDENTIAL_TYPES.includes(type));
   // Expected and not expected program length options.
-  const EXPECTED_PROGRAM_LENGTH = [
-    ProgramLengthOptions.WeeksToLessThanYear,
-    ProgramLengthOptions.OneToTwoYears,
-  ];
+  const EXPECTED_PROGRAM_LENGTH = [ProgramLengthOptions.WeeksToLessThanYear];
```

Comment:
This completely removes the test case for ProgramLengthOptions.OneToTwoYears as we split it into 2 now. Can we have the test case updated for your scenario? to validate the BCAG eligibility?